yamls/proto_nd_flow/workflows/util/tracklet_workflow.yaml

```diff
@@ -0,0 +1,23 @@
+# Generates higher-level event built data for charge data (i.e. tracklets)
```

Comment:
Perhaps a better place for this to live would be in `ndlar_flow/yamls/proto_nd_flow/reco/combined`?